webpack/assets/javascripts/react_app/components/Head/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+
+const Header = props => (
+  <Helmet>
+    {Object.entries(props).map(([TagName, value]) => (
+      <TagName key={TagName}>{value}</TagName>
```

Comment:
I assume it would work for simple cases like `<title>some text</title>` but won't allow you to do:
- `<base href="https://my-foreman-url.com" />`
- `<link rel="stylesheet" type="text/css" href="styles.css" />`
- `<meta charset="UTF-8" />`
- `<meta name="viewport" content="width=device-width, initial-scale=1.0" />`